ADJ-Internship/WebApp/wwwroot/css/PurchaseOrder.css

```diff
@@ -0,0 +1,28 @@
+.PO {
```

Comment:
can we modify the style of table in UIKit for consistency in the whole system?

Reply:
this is for tool testing purposes, Tấn will edit layout for whole system